demokratie-live/democracy-client
# Metadata for each Flavor

**Issue:** fixes #284 

**Description:**

Metadata for each Flavor

**How to test:**

We need a Deploy to test if this works 😅

- does not require a version - unless we want to specifically test it with a Deploy
- does not require a changelog
- does not require a development PR

Please check carefully!

**Checklist:**
